### PR TITLE
Add fastfilters2 recipe

### PR DESCRIPTION
**Pull request by emilmelnikov**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by emilmelnikov** [comment]
@conda-forge/help-python-c, ready for review!

**Comment by emilmelnikov** [comment]
@conda-forge-admin, please restart ci

**Comment by emilmelnikov** [comment]
@conda-forge-admin, please ping conda-forge/help-python-c

File: recipes/fastfilters2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [py<38]
```
Comment (xhochy) [inline recipes/fastfilters2/meta.yaml:14]: Not necessary anymore. We only build for py38+
```suggestion
```

File: recipes/fastfilters2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  script: {{ PYTHON }} -m pip install -vv .
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
```
Comment (xhochy) [inline recipes/fastfilters2/meta.yaml:20]:
```suggestion
    - {{ compiler("cxx") }}
    - make
```

File: recipes/fastfilters2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+{% set version = "2.0.0.dev0" %}
+{% set sha256 = "a671418a47079f165c18d4e8ef0415fa10dbfd4d7b458a384e856781864ed010" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (xhochy) [inline recipes/fastfilters2/meta.yaml:11]: Please put the sha directly here and avoid the indirection.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/fastfilters2/meta.yaml (hunk 2 above)
Comment (emilmelnikov) [reply recipes/fastfilters2/meta.yaml:18]: Specified `cmake` and `ninja` (for non-Windows platforms), [as suggested by scikit-build](https://scikit-build.readthedocs.io/en/latest/usage.html#example-of-setup-py-cmakelists-txt-and-pyproject-toml).

**Comment by emilmelnikov** [comment]
@conda-forge-admin, please ping conda-forge/help-python-c

Fixed PR suggestions, but used `cmake` and `ninja` instead of `make` in build dependencies.

**Comment by emilmelnikov** [comment]
@conda-forge-admin, please ping conda-forge/core

According to [the post-staging process](https://conda-forge.org/docs/maintainer/adding_pkgs.html#post-staging-process), [`fastfilters2-feedstock`](https://github.com/conda-forge/fastfilters2-feedstock) should be populated after a day or so, but it's currently empty. Did something went wrong on our side, or maybe this is a rate limiting issue?

**Pull request merged**